.gitignore

```diff
@@ -30,7 +30,7 @@ build/
 .signing/
 
 # Local configuration file (sdk path, etc)
-local.properties
+#local.properties
```

Comment:
You may have a merge conflict on the git ignore. I've updated it to be more exhaustive, including local.properties so it's fine.